### DISCUSSION TITLE
BMW Light Show

### DISCUSSION THREAD

**Discussion by cloudsapiens** [bimmerconnected/bimmer_connected#619] [Feature request] [answered]
Hi,

First of all, thanks for your work on this project. I'm working on a Tesla like light show with my BMW i4. I know, I will not have the same experience but still would like to play around with the lights that they kind of react on the bass or rhythm of a music. Any idea, how to overcome "too many requests" towards the BMW servers? How would you resolve it with the remote_services.trigger_remote_light_flash function?

Thanks and regards,
Norbert

**Comment by cloudsapiens** [comment]
I got following error:

Error triggering light flash on attempt 3: Remote service failed with state 'ExecutionState.ERROR'. Response: {'eventStatus': 'ERROR', 'errorDetails': {'title': 'Multiple requests sent to vehicle', 'description': 'Too many requests sent in a short time period. Please wait for couple of minutes to send request again.', 'presentationType': 'DIALOG', 'iconId': None, 'isRetriable': True, 'errorDetails': 'REFUSED_PLAY_PROTECTION'}}

**Comment by rikroe** [comment, marked as answer]
To be honest, I don't think this is possible.
BMW has traditionally been very conservative on allowed number of requests.
